flake.nix

```diff
@@ -1,40 +1,37 @@
 {
-  description = "Holochain Development Env";
+  description = "Flake for Holochain app development";
```

Comment:
Just checked as cached. Even if it's not, it's vastly smaller to build than the old Holonix because it doesn't include scaffolding or hc-launch

Reply:
This increased the workflow runtime from around [8 minutes](https://github.com/holochain/deepkey/actions/runs/10476565130) to [30+ minutes](https://github.com/holochain/deepkey/actions/runs/10478273225/job/29021249924?pr=31) While the components team is managing this project, I'd like to keep the current `flake.nix`.

Reply:
I would have liked the chance to tweak the workflow to see what the time is like with a cache in that case

Reply:
Sorry.  I was just in a hurry to merge to start bumping for v0.4.0-dev.18